pkg/security/rconfig/rconfig.go

```diff
@@ -27,14 +29,14 @@ type RCPolicyProvider struct {
 
 	client               *remote.Client
 	onNewPoliciesReadyCb func()
-	lastConfigs          []client.ConfigCWSDD
+	lastConfigs          map[string]state.ConfigCWSDD
 }
 
 var _ rules.PolicyProvider = (*RCPolicyProvider)(nil)
 
 // NewRCPolicyProvider returns a new Remote Config based policy provider
 func NewRCPolicyProvider(name string) (*RCPolicyProvider, error) {
-	c, err := remote.NewClient(name, []data.Product{data.ProductCWSDD})
```

Comment:
Can you rebase/merge with main ? We also added something to get the agent version in the provider, happy to discuss how to resolve a potential conflict if needed